### DISCUSSION TITLE
how to use @apply to target child of a certain class?

### DISCUSSION THREAD

**Discussion by laptopmutia** [tailwindlabs/tailwindcss#9196] [Help] [answered]
the case are like this 

if the button is under error div then the button color should be red else its should be indigo

```
@layer components {
  .error > .btn-primary {
    @apply bg-red-500
  }
  .btn-primary {
    @apply bg-indigo-500
  }
}
```

**Comment by wongjn** [comment, marked as answer]
Yes, that will work, as demonstrated by this [Tailwind Play](https://play.tailwindcss.com/wZxCxB2VYe).

**Reply by laptopmutia** [reply]
apparently it doesnt work for me because I use wrong syntax 

here are more example 

https://play.tailwindcss.com/xr8Cm9BUxf?file=css
